bitbots_utils/launch/tools.launch

```diff
@@ -1,8 +1,19 @@
 <?xml version="1.0"?>
 <launch>
-    <include file="$(find-pkg-share system_monitor)/launch/system_monitor.launch">
-        <arg name="required" value="false" />
-    </include>
+    <arg name="monitoring" default="true" description="true: start system monitor and udp bridge" />
+    <arg name="sim" default="false" description="Wether we are in a simulation environment" />
 
-    <include file="$(find-pkg-share udp_bridge)/launch/send.launch" />
+    <group>
```

Comment:
I'd like to have an `audio` argument (default true) to be able to manually disable the audio launch.